README.rst

```diff
@@ -216,55 +216,52 @@ In a nutshell:
+To create an application from a Cartridge template, run ``cartridge create`` in any directory:
 
-.. code-block:: bash
+..  code-block:: bash
 
     cartridge create [PATH] [flags]
 
-The following options (``[flags]``) are supported:
-
-.. // Please, update the doc in cli/commands on updating this section
-
-* ``--name string`` is an application name.
+Supported options (``[flags]``):
 
-* ``--from DIR`` is a path to the application template (see details below).
+.. // Please update the doc in cli/commands on updating this section
 
-* ``--template string`` is a name of application template to be used.
-  Currently only ``cartridge`` template is supported.
+* ``--name string`` – application name.
 
-Application is created in the ``<path>/<app-name>/`` directory.
+* ``--from DIR`` – path to the application template. For more about templates, see below.
 
-By default, ``cartridge`` template is used.
-It contains a simple Cartridge application with:
+* ``--template string`` – name of the application template to be used.
+  Only the template ``cartridge`` is supported at the time.
 
-* one custom role with an HTTP endpoint;
-* sample tests and basic test helpers;
-* files required for development (like ``.luacheckrc``).
+Your application will appear in the ``<path>/<app-name>/`` directory.
 
-If you have ``git`` installed, this will also set up a Git repository with the
-initial commit, tag it with
+If you have ``git`` installed, a Git repository with
+a ``.gitignore`` file will be also set up in the project root directory.
+The initial commit will be created, tagged with
 `version <https://www.tarantool.io/en/doc/latest/book/cartridge/cartridge_dev/#application-versioning>`_
-0.1.0, and add a ``.gitignore`` file to the project root.
+0.1.0.
 
-Let's take a closer look at the files inside the ``<app_name>/`` directory:
+^^^^^^^^^^^^^^^^^
+Project directory
+^^^^^^^^^^^^^^^^^
 
-* application files:
+Let's take a closer look at the files inside the ``<app_name>/`` directory.
 
-  * ``app/roles/custom-role.lua`` a sample
+* Application files:
+
+  * ``app/roles/custom-role.lua`` is a sample
     `custom role <https://www.tarantool.io/en/doc/latest/book/cartridge/cartridge_dev/#cluster-roles>`_
-    with simple HTTP API; can be enabled as ``app.roles.custom``
-  * ``<app_name>-scm-1.rockspec`` file where you can specify application
-    dependencies
-  * ``init.lua`` file which is the entry point for your application
-  * ``stateboard.init.lua`` file which is the entry point for the application
-    `stateboard <https://github.com/tarantool/cartridge/blob/master/rst/topics/failover.rst>`_
+    with a simple HTTP API. Can be enabled as ``app.roles.custom``.
+  * ``<app_name>-scm-1.rockspec`` contains application
+    dependencies.
+  * ``init.lua`` is the application entry point.
+  * ``stateboard.init.lua`` is the application
+    `stateboard <https://github.com/tarantool/cartridge/blob/master/rst/topics/failover.rst>`__ entry point.
```

Comment:
https://www.tarantool.io/en/doc/latest/book/cartridge/cartridge_dev/#failover-architecture
```suggestion
    :ref:`stateboard <cartridge-failover>` entry point.
```

Reply:
Also made this a `:doc:` link as per the [SG](https://www.tarantool.io/en/doc/latest/contributing/docs/markup/links/#linking-to-other-documentation-pages).